source/index.rst

```diff
@@ -1,40 +1,15 @@
-.. dst-doc documentation master file, created by
+.. Perun-node documentation master file, created by
    sphinx-quickstart on Thu May 17 17:20:50 2018.
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
-Welcome to the documentation of Direct State Transfer (DST) !
-=============================================================
+Welcome to Perun-node's documentation!
+======================================
 
-Direct State Transfer (DST) is an open source project that aims to
```

Comment:
should be changed to Perun

Reply:
See current master. This PR hasn't been rebased, yet...